spec/app_support/auto_logout_spec.rb

```diff
@@ -19,8 +44,8 @@
 
     it 'does not do anything' do
       expect { action.perform }.not_to change { reservation.reload.actual_end_at }
-      expect(order_detail.state).to eq('new')
-      expect(order_detail.order_status_id).to be_blank
+      expect { action.perform }.not_to change { order_detail.reload.state }
```

Comment:
This is a little weird to do `action.perform` multiple times. I think we should only be doing that once in the spec.